gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerFragment.java

```diff
@@ -138,6 +138,10 @@ private void onMapReady(MapAdapter map) {
     addFeatureBtn.setOnClickListener(
         __ -> homeScreenViewModel.onAddFeatureBtnClick(map.getCenter()));
     enableLocationLockBtn();
+    homeScreenViewModel
+        .getDownloadTileClicks()
+        .as(autoDisposable(this))
+        .subscribe(__ -> map.renderJsonLayer());
```

Comment:
Similarly, here we're "showing the offline basemap/imagery selector". Technically rendering of this GeoJSON layer is actually done by Maps API, and is an implementation detail.

Reply:
Good point. I'll excise the JSON from my names.